packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/node/ImageNode.tsx

```diff
@@ -89,16 +92,16 @@ export const ImageNode = memo(({ data, id, selected }: NodeProps<ImageNodeData>)
   useRefreshConnectionHandles(id, data.connectionHandles);
   return (
     <>
-      {data.nodeDescription?.userResizable && (
+      {data.nodeDescription?.userResizable && !readOnly ? (
```

Comment:
Why both remove the NodeResizer from the node and set it `shouldResize` property to false? I don't see the point of the change line 101

Reply:
Done